src/claim/assign-name/assign-name.js

```diff
@@ -0,0 +1,65 @@
+const Claim = require('../claim');
+const utils = require('../../utils');
+
+/**
+ * @param  {Uint32} Version
```

Comment:
Add a short description of the class please, not only parameters... Those parameters should go in the constructor comments